### PR TITLE
nf19: Add JSON config script and README.md.

### PR DESCRIPTION
**Pull request by haydenroche5**

### CODE REVIEW

File: 19-sign-and-door-tilt/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  "orientation": "face-down",
+  "temperature": 17.375,
+  "tilt": 1,
+  "voltage": 5.1286890300781243
+}
+```
+
+Let's break down the fields in the response:
+
+- `alert`: This will always be true when we get a `_motion.qo` note after issuing `card.motion.sync`.
```
Comment (haydenroche5) [inline 19-sign-and-door-tilt/README.md:77]: Not all of these fields are documented on dev.blues.io. I have done my best to determine what exactly they mean by reading the Notecard firmware code. @m-mcgowan your input here is appreciated. :)

Comment (m-mcgowan) [reply 19-sign-and-door-tilt/README.md:77]: If you've read the code then I think you know more than I do :-) although I thought `alert` meant a free-fall event.

**Review by m-mcgowan — APPROVED** [review]
LGTM. Can you please loop @tjvantoll into the API docs changes needed.

File: 19-sign-and-door-tilt/README.md (hunk 1 above)
Comment (haydenroche5) [reply 19-sign-and-door-tilt/README.md:77]: I read that somewhere, too, but the code that calls `motionAdd` when we've done `card.motion.sync` always passes true for the alert parameter:

```
if (config.AccelSync
        && 0 == timeSecsUntilDue(ALIGN_GPS, timeSecs(), motionLastSyncTimeSecs, config.AccelSyncMins * secs1Min)
        && motionThresholdMet(config.AccelSyncThreshold, config.AccelSyncCount)) {
    motionAdd(true);
    motionLastTrackTimeSecs = motionLastSyncTimeSecs = timeSecs();
    Sync("motion");
}
```

**Pull request merged**